src/System.ServiceModel.Primitives/tests/ServiceModel/ChannelFactoryTest.cs

```diff
@@ -118,10 +130,12 @@ public static void CreateChannel_Of_IRequestChannel_Using_BasicHttpBinding()
             if (factory != null)
             {
                 factory.Close();
+                Assert.Equal(CommunicationState.Closed, factory.State);
```

Comment:
Same comment -- this test is not testing factory.Close()'s affect on communication state.  I'd rather put that in a separate test if we want to verify it.